src/models/db/notification.js

```diff
@@ -1,18 +0,0 @@
-'use strict'
-
-const Model = require('five-bells-shared').Model
-const PersistentModelMixin = require('five-bells-shared').PersistentKnexModelMixin
-const knex = require('../../lib/knex').knex
-
-class Notification extends Model {
```

Comment:
Any future lookup and can basically be done on `transfers`. We send build the notification from the transfer object.